src/quak/__init__.py

```diff
@@ -18,8 +19,12 @@ def format(self, obj, include=None, exclude=None):
             # special case for duckdb relations
             if isinstance(obj, duckdb.DuckDBPyRelation):
                 obj = obj.arrow()
-            if is_arrow_ipc(obj) or is_dataframe_api_obj(obj):
+            elif has_pycapsule_stream_interface(obj):
                 obj = Widget(obj)
+            elif is_arrow_ipc(obj) or is_dataframe_api_obj(obj):
+                obj = Widget(obj)
+            else:
+                raise ValueError()
```

Comment:
These branches aren't mutually exclusive (we grab arrow from the duckdb so that it is captured in the next block), and we don't want to raise a `ValueError` for displaying stuff that can't be visualized with quak.

Right now

```python
"hello world"
```

Would raise a value error since it doesn't match any of the conditions.

Reply:
> we don't want to raise a `ValueError` for displaying stuff that can't be visualized with quak

Oh I see; this is probably the error I was hitting when I tried to `repr` something else (why I tried to unload quak)

Reply:
> we grab arrow from the duckdb

As a general note, why do you convert it to Arrow if you're passing it back to duckdb? I think duckdb can do replacement scans on `DuckDBPyRelation` objects.